lib/commands.py

```diff
@@ -117,8 +133,10 @@ def ssh(hostname_or_ip, cmd, check=True, simple_output=True, suppress_fingerprin
         if decode:
             output = output.decode()
 
-        errorcode_msg = "" if res.returncode == 0 else " - Got error code: %s" % res.returncode
-        logging.debug(f"[{hostname_or_ip}] {command}{errorcode_msg}{_ellide_log_lines(output_for_logs)}")
+        if not stream_output:
+            # Useless since output has already been printed
```

Comment:
Yes it has, in the case of `stream_output`.

Reply:
Your comment is inside the  `if not stream_output` block. One expects that it's about this case, not about the opposite case. That's why it's ambiguous.

Reply:
Suggestion: move the comment outside the block.
```
# Don't log the output in case of streaming since it has already been logged
if not stream_output:
    ...
```